askama/src/filters/alloc.rs

```diff
@@ -454,31 +454,6 @@ pub fn capitalize(s: impl fmt::Display) -> Result<String, fmt::Error> {
     capitalize(try_to_str!(s => buffer))
 }
 
-/// Count the words in that string.
```

Comment:
`wordcount` exists to get closer to parity with official `jinja`. I think we should keep it. Also, what would be the issue to have `values` passed as argument?

Reply:
Re-added and implemented with access to runtime values, though I'm not sure if anyone will ever use this filter. :)